ZXing.Net.Mobile.Forms.sln

```diff
@@ -29,6 +29,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ZXing.Net.Mobile.Forms.Wind
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ZXing.Net.Mobile.Forms.iOS", "Source\ZXing.Net.Mobile.Forms.iOS\ZXing.Net.Mobile.Forms.iOS.csproj", "{8370ADB4-229C-4DEC-908F-B1A8FC155523}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ZXing.Net.Mobile.macOS", "Source\ZXing.Net.Mobile.macOS\ZXing.Net.Mobile.macOS.csproj", "{A7E26AE0-12AF-4CFD-9B1E-3101347CC6F4}"
+EndProject
```

Comment:
maybe another new project ZXing.Net.Mobile.Forms.macOS is needed, to get Xamarin.Forms support?

Reply:
@knocte That would make sense, it seems like they define two classes: `ZXingScannerViewRenderer` and `ZXingBarcodeImageViewRenderer`, the former looks to be used by the scanner itself, while the latter looks to be used to render barcodes.

I'll get that thrown in here for macOS as well.